pkg/kubectl/scale.go

```diff
@@ -152,7 +152,7 @@ func (s *genericScaler) Scale(namespace, resourceName string, newSize uint, prec
 		err := wait.PollImmediate(
 			waitForReplicas.Interval,
 			waitForReplicas.Timeout,
-			scaleHasDesiredReplicas(s.scaleNamespacer, gr, resourceName, namespace, int32(newSize)))
+			ScaleHasDesiredReplicas(s.scaleNamespacer, gr, resourceName, namespace, int32(newSize)))
```

Comment:
let's not export this if we don't have to... it looks like the PR doesn't require it

Reply:
Actually, it is used in test/utils/update_resources.go